pages.fr/common/antibody.md

```diff
@@ -0,0 +1,16 @@
+# antibody
+
+> Le manager de module shell le plus rapide.
+> Plus d'informations : <https://getantibody.github.io>.
+
+- Regroupe tous modules pour un chargement statique :
+
+`antibody bundle < {{~/.zsh_modules.txt}} > {{~/.zsh_modules.sh}}`
+
+- Mets à jour tous les modules :
```

Comment:
```suggestion
- Met à jour tous les modules :
```

Reply:
Hi ! 

I must have an `s` at the end 😅 
![image](https://user-images.githubusercontent.com/9024389/179902518-f9e6c913-af97-497d-bb40-3c5caab2ecb5.png)

Reply:
OK, I think both are correct : `mets` for "(you) update" (_impératif présent_) and `met` is for "it updates" (_indicatif présent_). That's why I suggested `met`. :nerd_face: :sweat_smile: 

See https://www.larousse.fr/conjugaison/francais/mettre/6126 : 

> INDICATIF
> -Présent
> je mets
> tu mets
> il, elle met
> 
> IMPÉRATIF
> -Présent
> mets
> mettons
> mettez

Reply:
ooh that why !

But in the style-guide, they recommand imperative :p 

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood

Reply:
Well, imperative makes the most sense in English (and a lot of other language). But we should use what feels the most natural in a specific language.

Reply:
For me i rather like imperative than indicative. IMHO 😅